src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java

```diff
@@ -6,13 +6,15 @@
 import static com.codeborne.selenide.Condition.visible;
 import static com.codeborne.selenide.Selenide.$;
 
+import java.io.IOException;
 import java.time.Duration;
 import java.util.ArrayList;
 import java.util.Date;
 import java.util.Iterator;
 import java.util.List;
 import java.util.concurrent.atomic.AtomicInteger;
 
+import org.apache.commons.lang3.Range;
```

Comment:
at the moment this import is not used. I thought we wanted to check the timings with this helper.

Reply:
Done